### ISSUE TITLE
salt.states.win_network.managed cannot enable disabled network interface

### ISSUE THREAD

**Issue by kaluznyt** [saltstack/salt#45683]
### Description of Issue/Question
I'm trying to configure a state for managing windows 2012 r2. Basically what I want to achieve with this is to:
1. enable a disabled public network interface
2. install software (that needs public interface for downloading - that's mainly why I need step 1)
3. disable public network interface

### Setup
(Please provide relevant configs and/or SLS files (Be sure to remove sensitive info).)

top.sls

```
 server1:
    - enable-public-network-adapter
    - common-tools
    - disable-public-network-adapter

```

disable-public-network-adapter.sls
```
Disable public network adapter:
  network.managed:
    - name: Ethernet1
    - dns_proto: dhcp
    - ip_proto: dhcp
    - enabled: False
```

enable-public-network-adapter.sls    
```
Enable public network adapter:
  network.managed:
    - name: Ethernet1
    - enabled: True
    - dns_proto: dhcp
    - ip_proto: dhcp
    
```
 
### Steps to Reproduce Issue
(Include debug logs if possible and relevant.)

This is what I can see after running highstate on the server. 

```
----------
          ID: Enable public network adapter
    Function: network.managed
        Name: Ethernet1
      Result: False
     Comment: Unable to get current configuration for interface 'Ethernet1'
     Started: 11:01:18.268000
    Duration: 78.0 ms
     Changes:
----------
          ID: npp
    Function: pkg.installed
      Result: True
     Comment: All specified packages are already installed
     Started: 11:01:18.362000
    Duration: 8078.0 ms
     Changes:
----------
          ID: chrome
    Function: pkg.installed
      Result: True
     Comment: All specified packages are already installed
     Started: 11:01:26.440000
    Duration: 16.0 ms
     Changes:
----------
          ID: Disable public network adapter
    Function: network.managed
        Name: Ethernet1
      Result: True
     Comment: Interface 'Ethernet1' is up to date. (already disabled)
     Started: 11:01:26.456000
    Duration: 62.0 ms
     Changes:

Summary for XXXXX
------------
Succeeded: 3
Failed:    1
```

### Versions Report
(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)

Salt Version:
           Salt: 2017.7.2

Dependency Versions:
           cffi: 1.11.4
       cherrypy: Not Installed
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: 0.24.0
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: 2.18
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: 0.24.0
         Python: 2.7.12 (default, Nov 20 2017, 18:23:56)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: 0.9.0
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: Ubuntu 16.04 xenial
         locale: ISO-8859-1
        machine: x86_64
        release: 4.4.0-87-generic
         system: Linux
        version: Ubuntu 16.04 xenial

**Comment by gtmanfred** [comment]
@twangboy can you help with this when you get a minute?

Thanks,
Daniel

**Comment by kaluznyt** [comment]
I figured out that it probably didn't make much sense to include opposite states like I did, and rely on the order of events during state application. Therefore I just reconfigured the network so I don't need this any more. However, still would be nice to have the 'enabled' state applicable from 'disabled' for the network interfaces. Not sure what's the issue, but for disabled interfaces windows doesn't display too much on ipconfig.
